Apromore-Core-Components/Apromore-Portal/src/main/webapp/WEB-INF/web.xml

```diff
@@ -33,7 +33,6 @@
   <context-param>
     <param-name>contextClass</param-name>
     <param-value>org.eclipse.virgo.web.dm.ServerOsgiBundleXmlWebApplicationContext</param-value>
-    <!-- <param-value>org.springframework.osgi.web.context.support.OsgiBundleXmlWebApplicationContext</param-value> -->
```

Comment:
This commented-out code was left in there deliberately.  Making the Virgo context class continue to work under Karaf involves a hack I'd like to remove ASAP.  The commented-out version is the non-Virgo-specific alternative that doesn't require the hack.

Reply:
Just don't get it, a commented-out code was left in there _deliberately_? It cannot run, it is not being used. Did you mean it is kept there as a note for you to use it in the future.

Reply:
This is a configuration file, and that is an alternative configuration.  Please don't remove it.  I can write a lengthy comment warning people not to remove it and why, if you prefer.

Reply:
Ok, I'll add it back, and pls write some comment hint saying it's important to leave it there, for many people reading that.